proposals/0000-proposal-process.rst

```diff
@@ -50,87 +54,40 @@ Each proposal goes through the following stages:
    a file on GitHub), proposals can be made quickly without manual forking
    or cloning.
    
-2. Community members (including members of the proposal committee) will discuss
+2. Community members (including representatives of the projects who will
+   implement the proposed change) will discuss
    the proposal. The submitter is responsible for amending the specification to
    account for collected comments. It should be expected that the proposal will
    change during this process, which may last from days to months.
 
-3. When the author feels that the proposal is complete, and not being improved
-   by further discussion, they can submit the proposal to the proposal committee
-   for a decision.
-
-4. Within two (exceptionally three) weeks the committee will arrive at a
-   consensus as to whether the proposal in its submitted form meets a set of
-   criteria for inclusion (see below) while weighing the opinions expressed by
-   the community.
+3. When the author feels that all the relevant representatives are on board with
+   the proposal, and that the proposal would not be improved by further
+   discussion, they can write a comment indicating that they consider the
+   proposal accepted.
 
-   If the proposal is rejected then it will be sent back to the submitter along
-   with a rationale (referencing the criteria defined below). This is by no
-   means a final decision; the submitter may at any time amend and re-submit the
-   proposal to address the concerns of the committee.
-
-5. When the proposal is accepted the pull request will be merged and the
+4. One of the maintainers of this GitHub repository will then mark the proposal
+   as accepted by merging the pull request, and the
    document will be preserved in the proposals repository as a permanent
    specification for the feature.
    
-6. The author will create github issues on the appropriate ecosystem components,
+5. The author will create github issues on the appropriate ecosystem components,
    linking to the proposal to provide a place track the progress of the
    implementation.
 
-7. The author may choose to implement the proposal after acceptance, but she is
-   under no obligation to do so. (Of course, neither is anyone else.)
+6. The author may choose to implement the proposal after acceptance, but she is
+   under no obligation to do so. (Of course, neither is anyone else. This is why
+   it is important to get buy-in first.)
 
-8. During implementation, it is very likely that the specification will be refined.
+7. During implementation, it is very likely that the specification will be refined.
    The implementor should keep the specification up to date over the course of
    the implementation.
 
-9. If a proposal affects multiple components of the ecosystem and requires
-   coordination between them, there must be explicit buy-in from all affected
-   parties that they agree with the proposal.
```

Comment:
I deleted this paragraph because its contents is now emphasized elsewhere many times.